cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -203,6 +204,34 @@ def scale_ingress_controller():
         hookenv.log('Failed to scale ingress controllers. Will attempt again next update.')  # noqa
 
 
+@when('config.changed.labels', 'kubernetes-worker.config.created')
+def apply_node_labels():
```

Comment:
First run always sets the config.changed states, so this state as true on first run